### ISSUE TITLE
Calling cassandra nodes "Cluster Nodes" in UI is confusing

### ISSUE THREAD

**Issue by seglo** [Symantec/ambari-cassandra-service#5]
Just about all technologies provisioned through Ambari are "Clustered".  The Cassandra service plugin will reference Cassandra nodes simply as "Cluster Nodes", which is confusing within the context of Ambari.  I propose changing it to "C\* Nodes".

**Comment by ajak6** [comment]
yes needs to change the name to something better. C\* still does not properly explain what it actually maps to.

**Comment by seglo** [comment]
Maybe just "Cassandra Server" or "Cassandra Node" then.  Along with "Cassandra Client"
